tests/rich_env/serializers/test_base_model.py

```diff
@@ -4,27 +4,36 @@
 from typing import Any
 from unittest import TestCase
 
+from serialzy.serializers.base_model import ModelBaseSerializer
+
 from serialzy.api import Schema
 
 from serialzy.registry import DefaultSerializerRegistry
-from tests.rich_env.serializers.utils import serialize_and_deserialize
+from tests.rich_env.serializers.utils import serialize_and_deserialize, serialize_and_deserialize_with_meta
 
 
 class ModelBaseSerializerTests(TestCase):
     def initialize(self, module: str):
         self.module = module
         self.registry = DefaultSerializerRegistry()
 
-    def base_test(self, model: Any, expected_serializer):
-        serializer = self.registry.find_serializer_by_type(type(model))
-        deserialized_model = serialize_and_deserialize(serializer, model)
-
+    def _assert_model_serializer(self, serializer, expected_serializer):
```

Comment:
`-> None`  to turn on mypy here and downwards

Reply:
common style of this file forces absence of returning type declaration